### PR TITLE
Balder/use all memory we got.

### PR DESCRIPTION
**Pull request by baldersheim**
@geirst and @toregge PR

### CODE REVIEW

File: searchlib/src/vespa/searchlib/common/allocatedbitvector.cpp [hunk 1]
```diff
@@ -9,14 +9,25 @@ using vespalib::GenerationHeldBase;
...
 
-//////////////////////////////////////////////////////////////////////
-// Parameterized Constructor
-//////////////////////////////////////////////////////////////////////
+namespace {
+
+size_t computeCapacity(size_t capacity, size_t allocatedBytes) {
+    size_t possibleCapacity = (allocatedBytes * 8) - 1;
+    if (possibleCapacity < capacity) {
+       printf("capacity=%ld bytes=%ld possible=%ld\n", capacity, allocatedBytes, possibleCapacity);
```
Comment (geirst) [inline searchlib/src/vespa/searchlib/common/allocatedbitvector.cpp]: Change to debug logging or remove this.

**Review by geirst — APPROVED** [review]
👍 @toregge you must have a look as well. We now always call removeAllOldGenerations() in addDocs(), but I think that is ok.

File: searchlib/src/vespa/searchlib/common/allocatedbitvector.cpp (hunk 1 above)
Comment (baldersheim) [reply searchlib/src/vespa/searchlib/common/allocatedbitvector.cpp]: Removed.

**Comment by baldersheim** [comment]
Debug code removed.

**Comment by geirst** [comment]
@toregge ping

**Comment by baldersheim** [comment]
@toregge Ping

**Review by toregge — APPROVED** [review]

**Pull request merged**